### PR TITLE
Add skip sentry option

### PR DESCRIPTION
**Pull request by hirotnk**
@redink still need to test, but this is for early review and feedback.

### CODE REVIEW

**Comment by hirotnk** [comment]
still did not get a chance to test in a real system, but I'll test over the holiday season.

**Comment by hirotnk** [comment]
Worked on minimal tests: when passed the option `[metadata: [skip_sentry: true]]` to `Logger` as `Logger.error("error msg", [metadata: [skip_sentry: true]])` Sentry was skipped while if not, Sentry message was sent.

**Comment by hirotnk** [comment]
@redink Thank you for the review. I slightly modified the key for this option from `:metadata` to `:logger_sentry` to avoid possible conflicts with other backends. Please let me know if you want to modify anything

File: test/logger_sentry_test.exs [hunk 1]
```diff
@@ -50,7 +50,7 @@ defmodule LoggerSentryTest do
           {:warn, "warn_message"},
           {:error, "error_message"}
         ] do
-      :erlang.apply(Logger, :bare_log, [f, msg, [metadata: [skip_sentry: true]]])
+      :erlang.apply(Logger, :bare_log, [f, msg, [logger_sentry: [skip_sentry: true]]])
```
Comment (redink) [inline test/logger_sentry_test.exs]: `logger_sentry` is a more reasonable name here.

**Comment by redink** [comment]
I might want to change some variable names, but I'll do it with another pr. I think this pr is completely mergeable. 👍

**Comment by hirotnk** [comment]
@redink Thank you, updated the version.

**Comment by redink** [comment]
Published to hex https://hex.pm/packages/logger_sentry/0.5.0

**Comment by hirotnk** [comment]
Great, thank you @redink !

**Pull request merged**